### PR TITLE
Add ZioJsonBackend for Akka HTTP

### PR DESCRIPTION
**Pull request by jczuchnowski**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -266,8 +266,11 @@ lazy val akkaHttp = project
       "com.typesafe.akka"             %% "akka-http"                  % "10.2.4",
       "com.typesafe.akka"             %% "akka-serialization-jackson" % akkaVersion,
       "com.typesafe.akka"             %% "akka-stream"                % akkaVersion,
-      "de.heikoseeberger"             %% "akka-http-circe"            % "1.36.0" % Optional,
-      "de.heikoseeberger"             %% "akka-http-play-json"        % "1.36.0" % Optional,
+      "de.heikoseeberger"             %% "akka-http-circe"            % "1.36.0"   % Optional,
+      "de.heikoseeberger"             %% "akka-http-play-json"        % "1.36.0"   % Optional,
+      "de.heikoseeberger"             %% "akka-http-zio-json"         % "1.36.0"   % Optional,
+      "dev.zio"                       %% "zio-test"                   % zioVersion % "test",
```
Comment (ghostdogpr) [inline build.sbt:272]: The CI is currently calling `akkaHttp/compile`, can you change it to `akkaHttp/test`? (2 occurences in .circleci/config.yml)

**Review by ghostdogpr — COMMENTED** [review]
Looks great, just a tiny comment!

File: build.sbt (hunk 1 above)
Comment (ghostdogpr) [reply build.sbt:272]: I still don't see the test run, I think it needs `testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework"))` too

Comment (jczuchnowski) [reply build.sbt:273]: You're right. Should be fixed now.

**Review by ghostdogpr — APPROVED** [review]
Thanks! Great addition.

**Pull request merged**